orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/monitoreddeploy/NotifyDeployCompletedTask.java

```diff
@@ -45,6 +45,8 @@
       DeploymentMonitorDefinition monitorDefinition) {
     // TODO(mvulfson): actually populate the request data
     DeploymentCompletedRequest request = new DeploymentCompletedRequest(stage);
+    // request.setStatus(context.);
+    // request.setRolledback();
```

Comment:
Bester to delete code, rather than comment it out.

Reply:
@robzienert this is WIP, I'll edit the title of this PR.. Thanks for reviewing!

Reply:
Ah. If that's the case, try using the `Draft Pull Request` feature when you're opening the PR so it's a little more apparent.

Reply:
you shouldn't need anything here, instead add it to 
https://github.com/spinnaker/orca/blob/master/orca-deploymentmonitor/src/main/java/com/netflix/spinnaker/orca/deploymentmonitor/models/DeploymentCompletedRequest.java#L39